templates/accounts/resend_activation_mail.html

```diff
@@ -0,0 +1,29 @@
+{% extends 'layout.html' %}
+
+{% load static %}
+
+{% block title %} Jagrati | Resend activation mail {% endblock %}
+
+
+{% block content %}
+
+<div class="container pt-3" style="margin-top: 100px;">
+    <div class="text-center mt-4 p-0 p-sm-3" style="margin: auto; max-width: 500px;">
+        <h1 class="mt-2 font-weight-bolder" style="margin-bottom: 30px;">Resend Activation mail</h1>
+        <div class="form-outline">
+            <form method="post">
+                {% csrf_token %}
+                <input type="email" class="form-control" name="email" placeholder="Email"
+                    pattern="[a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\.[a-z0-9!#$%&'*+/=?^_`{|}~-]+)*@(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?"
+                    required />
+                <span class="form-error py-3 d-inline-block" style="color:red;">
```

Comment:
I see no use of `form-error` class here. Remove it, along with the `py-3` class (we'll be giving margin in the above and below element).

Reply:
But in the previous review, you told me to remove padding styles and add py-2 class

Reply:
If I remove py-3 class, this is the output.
![image](https://user-images.githubusercontent.com/61455329/112829516-dbb95e80-90ae-11eb-8ac9-6f6c85d1ffaa.png)

Reply:
Yeah, I might have told you that in the last review, but when I tested it out last time, I found adding margins in the components above and below the error message to be visually better than adding padding around the error message.

I would have changed that myself instead of asking you to do it but there were some important things to tell you in this review, adding docstrings being a major one, and I would have forgotten the changes I needed in the template in the meantime, so I added all those changes in the review itself, hoping you'll take something out of it too as on how to write clean code, as the main motive of this program is how much you learn, right?

For the above image, it will look fine once you add the suggested margins in the email text-box above and submit button below the error message.

Reply:
@garg3133 Yeah, Thank you! I am learning so much by solving this issue.
As you told, I added `my-2` class in the input element, removed `py-3` class from span element, removed `<br>`, added `mt-3 mb-5` in button element.
![image](https://user-images.githubusercontent.com/61455329/112834224-5b4a2c00-90b5-11eb-89e8-194dfb713797.png)
But the output is like this-
![image](https://user-images.githubusercontent.com/61455329/112834289-70bf5600-90b5-11eb-92c2-2a9a5d744d81.png)
But this is working fine - 
![image](https://user-images.githubusercontent.com/61455329/112834395-8df42480-90b5-11eb-98cc-aa88cfd7b03d.png)

Reply:
Maybe it's happening because you remove the `d-inline-block` class from `span`.

Reply:
@garg3133 Even after I add `d-inline-block`, I am getting the same output
![image](https://user-images.githubusercontent.com/61455329/112845002-fc3ee400-90c1-11eb-895b-f11848711cb0.png)

Reply:
Oh, then try using `d-bock` only. I will definitely solve the problem.

Reply:
Solved.